src/main/java/kitchenpos/order/common/domain/service/OrderServiceImpl.java

```diff
@@ -72,7 +72,7 @@ public Order create(final Order request) {
             if (!menu.isDisplayed()) {
                 throw new IllegalStateException();
             }
-            if (menu.getPrice().compareTo(orderLineItemRequest.getPrice()) != 0) {
+            if (menu.getPrice().price().compareTo(orderLineItemRequest.getPrice()) != 0) {
```

Comment:
디미터법칙에 대해서 학습해 보면 좋을 것 같아요
```suggestion
            if (menu.getPrice().compareTo(orderLineItemRequest.getPrice()) != 0) {
```

Reply:
감사합니다! 참고해볼게요!!